src/main/java/edu/usf/cutr/gtfsrtvalidator/api/resource/GtfsRtFeed.java

```diff
@@ -259,16 +259,24 @@ public Response getMonitorData(
 
     // Returns feed message for a requested iteration
      @GET
-     @Path("/{iterationId : \\d+}/feedMessage")
+     @Path("/{iterationId : -?\\d+}/{gtfsRtId : \\d+}/feedMessage")
      @Produces(MediaType.APPLICATION_JSON)
      public String getFeedMessage(
-             @PathParam("iterationId") int iterationId) {
+             @PathParam("iterationId") int iterationId,
```

Comment:
I think these would both make more sense as `@QueryParam`?  Let me know what you think.

Reply:
I made the related changes in the review API parameters issue #216

Reply:
I am going to raise a pull request for #216 as soon as we merge this request

Reply:
Sounds good, we'll change the parameters in a separate pull request.